sites/partners/src/listings/PaperListingDetails/sections/DetailListingPhoto.tsx

```diff
@@ -18,9 +18,12 @@ const DetailListingPhoto = () => {
   if (listing.image == null && listing.assets.length > 0) {
     listingFormPhoto = listing.assets.find((asset) => asset.label == "building")
   }
-  const listingPhotoUrl = /https?:\/\//.exec(listingFormPhoto.fileId)
-    ? listingFormPhoto.fileId
-    : cloudinaryUrlFromId(listingFormPhoto.fileId)
+
+  const listingPhotoUrl = listingFormPhoto
+    ? /https?:\/\//.exec(listingFormPhoto.fileId)
+      ? listingFormPhoto.fileId
+      : cloudinaryUrlFromId(listingFormPhoto.fileId)
```

Comment:
I know this was already here, but regex.test might be better used here instead of exec.